docs/dev/build-instructions/build_linux.md

```diff
@@ -37,11 +39,42 @@ If you intend to include grpc-support for the Mumble server (murmur), you also h
 The dependence on `g++-multilib` only applies if you are on a 64bit system and want to cross-compile overlay support for 32bit applications as well
 (which is enabled by default). If you don't do this (`-Doverlay-xcompile=OFF` when invoking cmake), you also don't have to install `g++-multilib`.
 
-If you are using a distribution that is not based on Ubuntu, you probably have to change the command accordingly.
-
 You will need `cmake` **v3.15 or later**. If the version shipped by your distribution is not recent enough, you can install a recent one via the
 [official PPA](https://apt.kitware.com/) or from the [linked page](https://cmake.org/download/).
+</details>
+
+
+<details>
+  <summary>CentOS 8</summary>
```

Comment:
```suggestion
  <summary><b>CentOS 8</b></summary>
```
Btw: Are these commands expected to be different for different CentOS versions? If not, I think I'd tend to remove the version number :thinking:

Reply:
Cent OS stream uses dnf at the moment. Cent OS 7 used yum to install packages.